src/sentry/lang/native/symbolicator.py

```diff
@@ -19,7 +19,7 @@
 from sentry.net.http import Session
 from sentry.tasks.store import RetrySymbolication
 
-MAX_ATTEMPTS = 3
+MAX_ATTEMPTS = 2
```

Comment:
Let's maybe leave this and the `wait` backoff, since we might otherwise risk failing too many events.